UPGRADE.md

```diff
@@ -13,7 +13,7 @@ Sulu now uses Webpack 5 to build the administration interface application. To en
 - `optimize-css-assets-webpack-plugin` was removed replaced by `css-minimizer-webpack-plugin`: `^4.2.2`
 - `clean-webpack-plugin` was removed and replaced by `clean: true` webpack output option
 - `webpack-clean-obsolete-chunks` was removed and replaced by `clean: true` webpack output option
-- `is-email` was removed and replaced by `sulu-admin-bundle/utils/Email/validateEmail` method 
+- `is-email` was removed and replaced by `sulu-admin-bundle/utils/Email/validateEmail` method
```

Comment:
please revert the unnecessary changes to keep here the git histories correctly of this lines

Reply:
Removed the BC break so that changing the `UPGRADE.md` wasn't necessary anymore.